index.bs

```diff
@@ -90,6 +93,8 @@ A <dfn export id=file>file entry</dfn> additionally consists of
 a <dfn for="file entry">lock</dfn> (a string that may exclusively be "`open`", "`taken-exclusive`" or "`taken-shared`")
 and a <dfn for="file entry">shared lock count</dfn> (a number representing the number shared locks that are taken at a given point in time).
 
+<!-- TODO: Consider tying a locks to a path, to support moves -->
```

Comment:
Unless we're okay with locking entries which don't (yet) exist...

Reply:
discussed in other threads. Posting all operations to the file system lock queue (see #95) would resolve this